inbm-vision/node-agent/node/constant.py

```diff
@@ -99,3 +99,6 @@
 CONFIG_HEARTBEAT_RESPONSE_SECS_UPPER_LIMIT = 1800
 CONFIG_HEARTBEAT_RESPONSE_SECS_LOWER_LIMIT = 90
 CONFIG_HEARTBEAT_RESPONSE_SECS_DEFAULT = 300
+
+# Var directory
+VAR_DIR = '/var'
```

Comment:
Looks like this file is missing the new line.  I can add that so we can merge today.